packages/gluestick/transpile.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+transpile () {
+  local dirname=$1
+  local tmpdirname=$dirname"-tmp"
+
+  $(npm bin)/babel $dirname --presets=react,es2015,stage-0 --plugins=transform-decorators-legacy,transform-flow-strip-types -d $tmpdirname && rm -rf $dirname && mv $tmpdirname $dirname & \
```

Comment:
Yeah, true - it was like this only for testing purposes, that's why it's under `wip`.  We are still thinking, about prepublish transpilation and we have mixed feelings. Finally, if we'll decide to go for this solution there should be some other output directory for transpiled code 😇.